docs/running-node-ansible/prerequisites.md

```diff
@@ -0,0 +1,74 @@
+# Running a DCLedger Node using Ansible (Prerequisites)
+
+This document describes in how to:
+
+* configure different types of DCLedger nodes: genesis, validator and observer
+* create a node administrator (or admin) account as a necessary part of validator and genesis node configuration
+
+## Requirements (local machine)
+
+### 1. Clone [DCL][5] project
+```bash
+git clone https://github.com/zigbee-alliance/distributed-compliance-ledger.git
+```
+
+### 2. Install `Python` and `Pip`
+```bash
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends python3
+sudo apt install python3-pip python3-testresources
```

Comment:
Shouldn't be better to install all the packages in a single instruction?
```suggestion
sudo apt-get install -y --no-install-recommends python3 python3-pip python3-testresources
```

BTW, can we include `python3-testresources` as part of the `requirements.txt` file?